README.md

```diff
@@ -111,22 +136,115 @@ The Customer Data Cloud Accelerator allows reading, working locally and deployin
 
-### Local Live Preview <a id="features-local-preview"></a>
+### How to configure the use of features on the file cdc-accelerator.json
+
+On the cdc-accelerator.json file, there are two mandatory properties that the user has to fill, the "source" and "deploy".
+They both will have an array of objects that will contain the apiKeys that are related to the sites that we want to use in the project and optionally it will have the features, for example:
+
+```sh
+{
+  "source":[
+    {
+    "apiKey":"1_B12AD0AISOPAD",
+    "features": ["Schema","PermissionGroup","WebSdk"]
+    }
+  ]
+}
+```
+
+### An empty array will mean no feature will be available, for example:
+
+```sh
+{
+  "source":[
+    {
+    "apiKey":"1_B12AD0AISOPAD",
+    "features": []
+    }
+  ]
+}
+```
+
+### How to use the feature specific commands
 
-Local live preview allows the consultant to see and test the web screen sets and email templates on the local environment, before deploying them to the CDC console.
+Using the feature specific command lets the user run a specific feature (Schema, PermissionGroups, WebSdk, WebScreenSets,EmailTemplates...) instead of running all of them when doing an operation (init,reset,build,deploy).
+To use them, simply write on the terminal
 
-#### Web Screen-Sets <a id="features-local-preview-web-screen-sets"></a>
+```sh
+npm run <operation> -- -f <feature>
+```
+
+For example:
+
+```sh
+npm run init -- -f Schema
+```
+
+In this example the user is only going to run the feature Schema when running the operation init, the feature name can be replaced by any other feature (Email Templates, WebScreenSet, PermissionGroup, WebSdk...).
+To show all the possible commands, the user can write simply
+
+## Preview
+
+The preview mode is a feature that allows the user to see and test the changes in the local environment, without the need to deploy the data to the customer data cloud console.
+
+### How to use filters on preview
+
+The filter is applied on the "src/index.html" file, that will filter the screens that the user will choose to see by using the apiKeys that are configured on the configuration file cdc-accelerator.json, for example:
+
+```sh
+ [{
+    apiKey: '1_2ABCDEFGHI345',
+    screens: [{ screenSetID: 'PreferencesCenter-ProfileUpdate', screenID: 'gigya-update-profile-screen' }],
+    emails: [ { emailID: 'codeVerification', languages: ['en'] } ]
+  }]
+```
+
+### Using the filter on more than one apiKey
+
+```sh
+[{
+    apiKey: '1_2ABCDEFGHI345',
+    screens: [
+      { screenSetID: 'PreferencesCenter-ProfileUpdate', screenID: 'gigya-update-profile-screen' },
+      { screenSetID: 'PreferencesCenter-Landing', screenID: 'gigya-login-screen' },
+    ],
+    emails: []
+  },
+  {
+    apiKey: '1_3AS9DJAKSLA12',
+    emails: [{ emailID: 'doubleOptIn', languages: ['ar', 'en', 'pt-br'] } ]
+  }]
+```
+
+### Adding the filter to all the ApiKeys that do not have already a specific filter that are on the configuration file
```

Comment:
What about the title to be "Adding a filter to all the Api keys that do not have already a specific filter"?